Samples/LambdaLang/LambdaExpressionEvaluator.cs

```diff
@@ -0,0 +1,70 @@
+using LambdaLang.Language;
```

Comment:
As we don't want to sample project is not used in test project, recovery it to its original structure, this evaluate expression should not be abstracted